### PR TITLE
Use static runc binary in CircleCI

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
We can finally use the static binary because it contains all necessary build tags.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: scripts/circle-setup [hunk 1]
```diff
@@ -116,15 +116,10 @@ install_cni_plugins() {
...
-    pushd "$GOPATH"/src/$RUNC_PATH
-    git checkout "${VERSIONS["runc"]}"
-    make BUILDTAGS="apparmor selinux seccomp"
-    sudo cp runc /usr/sbin
-    popd
+    URL=https://github.com/opencontainers/runc/releases/download
+    BINARY=/usr/sbin/runc
+    sudo wget -O "$BINARY" $URL/"${VERSIONS["runc"]}"/runc.amd64
+    sudo chmod +x "$BINARY"
     runc --version
```
Comment (TomSweeneyRedHat) [inline scripts/circle-setup]: should you replace runc with $BINARY here?

**Comment by TomSweeneyRedHat** [comment]
one question, otherwise LGTM

File: scripts/circle-setup (hunk 1 above)
Comment (saschagrunert) [reply scripts/circle-setup]: Hm, no this is more just a verification the `runc` is in `$PATH` :)

Comment (dougsland) [reply scripts/circle-setup]: For security reasons, I would suggest compare sha256sum from the file before using it.

$ wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc10/runc.amd64
$ wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc10/runc.sha256sum

$ sha256sum runc.amd64 
a01afd5ff47d5a2a96bea3a871fb445b432f90c249a8a5d5239b05fe0d5bee4a  runc.amd64

$ cat runc.sha256sum | grep -w a01afd5ff47d5a2a96bea3a871fb445b432f90c249a8a5d5239b05fe0d5bee4a

a01afd5ff47d5a2a96bea3a871fb445b432f90c249a8a5d5239b05fe0d5bee4a  runc.amd64

**Comment by rhatdan** [comment]
@mrunalp @haircommander @umohnani8 PTAL

File: scripts/circle-setup (hunk 1 above)
Comment (saschagrunert) [reply scripts/circle-setup]: Yes, I add this as well.

**Pull request merged**